cubic-server/types.cpp

```diff
@@ -1,6 +1,5 @@
 #include "types.hpp"
 #include "utility/PseudoRandomGenerator.hpp"
-#include <cstdint>
```

Comment:
Why? We use some of cstdint's types in that file...

Reply:
It's already included in types.hpp

Reply:
It doesn't matter, as we have include guards, but aight